### PR TITLE
frp: update to 0.58.1.

### PR DESCRIPTION
**Pull request by Suyun114**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

**Comment by Suyun114** [comment]
How can I trigger the CI?

**Comment by classabbyamp** [comment]
for first-time contributors, it's manually triggered

File: srcpkgs/frp/template [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-version=0.37.1
-revision=3
+version=0.58.1
+revision=1
 build_style=go
 go_import_path=github.com/fatedier/frp
 go_package="${go_import_path}/cmd/frpc ${go_import_path}/cmd/frps"
 short_desc="Fast reverse proxy"
-maintainer="Anachron <gith@cron.world>"
+maintainer="Suyun <suyun@aosc.io>"
```
Comment (classabbyamp) [inline srcpkgs/frp/template:9]: revert this line, @Anachron is still the maintainer

Comment (Suyun114) [reply srcpkgs/frp/template]: i am sorry for the misunderstanding

**Pull request merged**